kernel/README.md

```diff
@@ -5,21 +5,21 @@
 When updating kernel to the new version, import proper defaults with:
 
 ```sh
-make kernel-olddefconfig
+make kernel-olddefconfig USERNAME=rsmitty
 ```
 
 If you want to update for a specific architecture only, use:
 
 ```sh
-make kernel-olddefconfig PLATFORM=linux/arm64
+make kernel-olddefconfig USERNAME=rsmitty PLATFORM=linux/arm64
 ```
 
 ## Customizing the kernel
 
 Run another target to get into `menuconfig`:
 
 ```sh
-make kernel-menuconfig
+make kernel-menuconfig USERNAME=rsmitty
```

Comment:
.

Reply:
what does `.` mean? :)

Reply:
Didn't know what else to say, but thought it would get your attention :smile:

Reply:
I thought this was a typo.

Reply:
just docs, better to override